test/comment-regex.js

```diff
@@ -51,15 +69,93 @@ test('comment regex new spec - #', function (t) {
     '\t/*# ', // multi line style with leading tab
     '/*# ',   // multi line style with leading text
   ].forEach(function (x) { 
-    t.ok(comment(x, ''), 'matches ' + x) 
-    t.ok(commentWithCharSet(x, ''), 'matches ' + x + ' with charset') 
-    t.ok(commentWithCharSet(x, '', '='), 'matches ' + x + ' with charset')
+    t.ok(comment(x, '', convert.commentRegex), 'matches ' + x)
```

Comment:
ditto here a bunch of stuff changed .. maybe I'm just missing something and this is smaller than appears, but I still don't understand why this was needed

Reply:
Oh yeah, this changed because different regexps were being passed to test them all, this can be reduced back to the original because it is one regexp again

Reply:
Reduced the diff!